### PR TITLE
Add sneat app

### PR DESCRIPTION
**Pull request by xcarpentier**
Add sneat app in all apps.

### CODE REVIEW

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -486,6 +486,12 @@ var apps = [
     author: 'Dobbit Co., Ltd.'
   },
   {
+    name: 'sneat: réservez les meilleurs restaurants de Paris',
+    icon: 'http://a3.mzstatic.com/eu/r30/Purple49/v4/71/71/df/7171df47-6e03-8619-19a8-07f52186b0ed/icon175x175.jpeg',
+    linkAppStore: 'https://itunes.apple.com/fr/app/sneat-reservez-les-meilleurs/id1062510079?l=en&mt=8',
```
Comment (chirag04) [inline website/src/react-native/showcase.js]: use link key instead of linkAppStore. linkAppStore is when you have android and ios both.

Comment (xcarpentier) [inline website/src/react-native/showcase.js]: @chirag04 thanks !
It's fixed.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
this got merged buy I am pretty sure the last update was ignored

**Comment by xcarpentier** [comment]
@bestander ok thanks !

**Pull request closed without merging**